### DISCUSSION TITLE
After upload file in the form, the workflow does not work

### DISCUSSION THREAD

**Discussion by horizontech-ernest** [AOT-Technologies/forms-flow-ai#1353] [Q&A] [answered]
[form and workflow.zip](https://github.com/AOT-Technologies/forms-flow-ai/files/11098874/form.and.workflow.zip)
Hi, I have made the form and workflow (inside the attached zip).

In the form, I have added a file component to allow applicant to upload the file.

However, I found that if I does not upload file in the form, the entire workflow works well. But if I upload the file in the form, the workflow will not proceed after the application is created (e.g. attached pic). May I know how can I solve this issue?

Remark: I have done any specific set up for the file component (just selected the storage type)

![image](https://user-images.githubusercontent.com/126135607/228493482-41562bd0-b361-4ff0-acda-658501b14666.png)

Updated:

I found that the error is that "was aborted: ERROR: value too long for type character varying(4000)  Call getNextException to see other errors in the batch." How should I solve it?
<img width="1512" alt="Screenshot 2023-03-29 at 7 34 29 PM" src="https://user-images.githubusercontent.com/126135607/228523220-af36c124-9b91-4800-acbd-5d0f3086cc14.png">

<img width="1512" alt="Screenshot 2023-03-29 at 7 34 37 PM" src="https://user-images.githubusercontent.com/126135607/228523256-f3ff0ef7-5e68-48e8-a3fb-c53df4705578.png">

**Comment by SnehaSuresh-aot** [comment, marked as answer]
Hello @horizontech-ernest,
This issue may be due to the large file size you uploaded. Have you tried uploading the file with smaller size? Or else you can configure the buffer size by setting the environment variable `DATA_BUFFER_SIZE`  to desired value. Default value is 2 MB. 
https://github.com/AOT-Technologies/forms-flow-ai/blob/eb1905c9894ced0db0becbb1b6cc325061cd8a75/forms-flow-bpm/sample.env#L68

**Reply by xalcolm1** [reply]
Hi @SnehaSuresh-aot I tried uploading a file with only 3kb and I'm still getting this error and also after increasing the DATA_BUFFER_SIZE.
 I was trying with too big files initially, is it possible that something got corrupted in the Camunda database?
Thanks in advance for any pointers and tips
